Automattic/newspack-plugin
# feat(ras-acc) add welcome email template for checkout registration

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://app.asana.com/0/1207385073694403/1205250932343413/f

This PR is partnered with the following main plugin PR: https://github.com/Automattic/newspack-blocks/pull/1754

This adds a new welcome email template for readers who register via checkout as part of RAS-ACC.

![Screenshot 2024-05-31 at 17 32 46](https://github.com/Automattic/newspack-plugin/assets/17905991/6266d18a-ff62-42c9-8781-2ea2733f19e3)

### How to test the changes in this Pull Request:

Before testing, be sure to check out the changes in https://github.com/Automattic/newspack-blocks/pull/1754

- As admin ensure the new welcome email template as well as the receipt template are enabled via Newspack > Reader Revenue > Emails

![Screenshot 2024-05-31 at 17 34 28](https://github.com/Automattic/newspack-plugin/assets/17905991/b1d28a94-07f5-40d4-8699-bf18cf49ff0c)

- In an incognito window as a logged out reader, make a donation to the test site via a donation block
- Complete checkout, creating an account in the process, and confirm you receive the new welcome email after checkout (pictured above)
- Refresh the page, and make another donation logged in as the same reader from the previous step and confirm you receive the receipt email this time. (They are very similar but the receipt email does not have any welcome text and does not contain a button leading to the site)
- As admin, disable the welcome email and ensure the receipt template is still enabled
- Again as a reader, **log out** and make another donation.
- Confirm you do not receive the welcome email and instead receive the receipt email.

### Other information:
